Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/transform/audit/AdminDistTransforms.java

```diff
@@ -100,8 +100,7 @@ protected LogEventRequestType getLogEventRequestType(EDXLDistribution body, Asse
 
         result.setAuditMessage(auditMsg);
         result.setDirection(direction);
-        result.setInterface(_interface);
-
+        result.setRemoteHCID(HomeCommunityMap.formatHomeCommunityId(communityId));
```

Comment:
getAdminDistributionMessageCommunityID() is still used to get remoteHCID. However I will remove interface from this method and only look up based on direction.